Tools/CISupport/ews-build/events.py

```diff
@@ -260,7 +260,11 @@ def stepStarted(self, key, step):
             build['properties'] = yield self.master.db.builds.getBuildProperties(step.get('buildid'))
 
         # We need to force the defered properties to resolve
-        if build['properties'].get('github.number'):
```

Comment:
Better to have this list of steps in a separate variable.
Also, you can copy the list from https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-app/ews/views/statusbubble.py#L65
and add a fixme to remove either one of it in long-term.